src/Cawbird.vala

```diff
@@ -94,13 +94,23 @@ public class Cawbird : Adw.Application {
    * Initialize the client and open the first window.
    */
   protected override void activate () {
+    Backend.Client client;
     // Initializes the backend client
-    var client = new Backend.Client (Config.PROJECT_NAME,
-                                     "https://github.com/CodedOre/NewCaw",
-                                     "cawbird://authenticate");
+    try {
+      client = new Backend.Client("id",
```

Comment:
```suggestion
      client = new Backend.Client(Config.APPLICATION_ID,
```

This should use the right id. Otherwise `KeyStorage` will save the access tokens for an application named `id`.

Reply:
Ooops. Not sure how I managed to get that in there. Good catch!